CONTRIBUTING.md

```diff
@@ -242,7 +242,11 @@ To keep naming patterns consistent across the project, naming patterns are enfor
 - Variable assigned in a package's global scope that is preconfigured with a default set of values MUST use `Default` as the prefix. For example:
   - `var DefaultMarshallers = map[string]pdata.Marshallers{...}` is defined with an exporters package that allows for converting an encoding name,
     `zipkin`, and return the preconfigured marshaller to be used in the export process.
-
+- Types that are specific to a signal MUST be worded with the signal used as an adjective, i.e. `SignalType`. For example:
+  - `type TracesSink interface {...}`
+- Functions that are specific to a signal MUST be worded with the signal type as a direct object, i.e. `VerbSignal`. For example:
+  - `func ConsumeTraces(...) {...}`
+  - `func CreateTracesToTracesFunc(...) {...}`
```

Comment:
This example is somewhat confusing for the direction above as it is `VerbSignalToSignalFunc`, maybe this requires an additional guideline around naming func types

Reply:
I think it's mostly because `VerbSignalToSignalFunc` doesn't directly follow the pattern laid out above. I tried to expand on this more to cover these cases, let me know what you think.